loggly/loggly.go

```diff
@@ -76,59 +79,80 @@ func (l *Adapter) Stream(logstream chan *router.Message) {
 			ContainerImage:    m.Container.Config.Image,
 			ContainerHostname: m.Container.Config.Hostname,
 		}
-
-		l.SendMessage(msg)
+		a.enquequeMessage(msg)
 	}
 }
 
-// SendMessage handles creating and sending a request to Loggly. Any errors
-// that occur during that process are bubbled up to the caller
-func (l *Adapter) SendMessage(msg logglyMessage) {
-	js, err := json.Marshal(msg)
+// dispatcher sends bulks of bufferd logs every 2 seconds after a request to loggly
+func (a *Adapter) dispatcher() {
```

Comment:
dispatcher is an opaque name when the API is called enqueueMessage. can you call it dispatchMessages?